usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -533,9 +533,8 @@ static CK_RV cleanse_attribute(TEMPLATE *template,
     CK_RV rc;
```

Comment:
rc looks unused now.

Reply:
Yes, you're right ... I will remove this line.